.eslintignore

```diff
@@ -8,4 +8,5 @@ coverage/*
 /src/main/public/
 # custom definition files
 /src/main/types/
-gulpfile.js
\ No newline at end of file
+gulpfile.js
+mochawesome-report
```

Comment:
What about
coverage-report also?

Reply:
doesn't cause any issue at the moment .. does lint complain in your local?

Reply:
I could of sworn it did but just tested again and no it does not. We can add it in if we find linter has problems with it.